### DISCUSSION TITLE
NAN values- dynamic smagorinsky and wale models

### DISCUSSION THREAD

**Discussion by dkrgajjela** [xcompact3d/Incompact3d#183] [Q&A] [answered]
Hi,
I'm trying to use dynamic smagorinsky model for my case. I'm getting NAN values in the first time step itself whatever may be the grid size and time step. I've also tried the wale model which is also giving NAN results. I've taken the **maxdsmagcst** as 0.17 for dynamic smagorinsky model and **walecst** as 0.325 for wale model. Can some please explain where I'm doing the mistake.
Thank You.

**Comment by slaizet** [comment, marked as answer]
Hi, 
Thank you for your message. Please provide all necessary information to explain you problem. We do not even know what is the flow set up. I will only add that we have only tested those models for very few set-up so they might not work for your set-up.
Best,
Sylvain

**Reply by dkrgajjela** [reply]
Thank you for the reply @slaizet . I've run the two turbine example case and I've attached the input file. I've also attached the log file with the error.
[input.txt](https://github.com/xcompact3d/Incompact3d/files/11531589/input.txt)
[log.txt](https://github.com/xcompact3d/Incompact3d/files/11531621/log.txt)
Thank You.

**Reply by slaizet** [reply]
There is no reasons to use an explicit LES model in this set-up as the implicit LES is working great. The WALE model is for when you have a wall, which is not the case for this set-up. The dynamic Smagorinsky model is super expensive and requires a lot of tuning (for instance how to compute the Smag constant, filtering parameters). If you -really- want to use an explicit LES model, then you should use the standard Smag model (as there is no wall). As far as I am concerned, I would not want to spend time trying to use an explicit LES model in this set-up :-)
Best,
Sylvain
